x-pack/plugins/security_solution/public/flyout/left/components/correlations_details_alerts_table.tsx

```diff
@@ -148,18 +148,19 @@ export const CorrelationsDetailsAlertsTable: FC<CorrelationsDetailsAlertsTablePr
       header={{
         title,
         iconType: 'warning',
-        headerContent: (
-          <div data-test-subj={`${dataTestSubj}InvestigateInTimeline`}>
-            <InvestigateInTimelineButton
-              dataProviders={dataProviders}
-              filters={filters}
-              asEmptyButton
-              iconType="timeline"
-            >
-              {ACTION_INVESTIGATE_IN_TIMELINE}
-            </InvestigateInTimelineButton>
-          </div>
-        ),
+        headerContent:
+          alertIds && alertIds.length && alertIds.length > 0 ? (
```

Comment:
@PhilippeOberti adding a check here to only show investigate in timeline when id length > 0. Feel free to revert if this is considered code change

Reply:
this is good, thanks for fixing this!